packages/acceptit-Core.package/ACParser.class/instance/parse.class.noPattern.context.notifying.ifFail..st

```diff
@@ -17,6 +17,5 @@ parse: sourceStream class: cls noPattern: pat context: ctx notifying: req ifFail
 		parse: codeToParse
 		class: cls
 		noPattern: pat
-		context: ctx
```

Comment:
What does this even do?

Reply:
The method that was previously called doesn't exist, I'm not sure if it was just a copy paste error or something, but this is the actual parse message that should be used.